components/brave_wallet/common/eth_sign_typed_data_helper.cc

```diff
@@ -71,8 +66,8 @@ void EthSignTypedDataHelper::FindAllDependencyTypes(
     }
     const std::string* type = field.GetDict().FindString("type");
     if (type) {
-      auto type_split = base::SplitString(*type, "[", base::KEEP_WHITESPACE,
-                                          base::SPLIT_WANT_ALL);
+      auto type_split = base::SplitStringPiece(
```

Comment:
```suggestion
      const auto type_split = base::SplitStringPiece(
```
nit: `const std::vector<std::string_view>`